install-and-configure/install/provider-installations/openshift-kubecost-install.md

```diff
@@ -42,7 +42,7 @@ Install Kubecost using OpenShift specific values. Note that the below command fe
 
 ```sh
 helm upgrade --install kubecost kubecost/cost-analyzer -n kubecost --create-namespace \
--f https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/develop/cost-analyzer/values-openshift.yaml
+-f https://raw.githubusercontent.com/kubecost/cost-analyzer-helm-chart/<$VERSION>/cost-analyzer/values-openshift.yaml
```

Comment:
The drawback to doing this is that now this link will no longer be checked by the CI's link-checker. What happens if we someday decide to rename or delete `values-openshift.yaml`?

Although referencing `develop` branch comes with drawbacks, I still think it's more valuable than putting `<$VERSION> here. @mittal-ishaan @chipzoller thoughts?

Reply:
We can develop. But we can also start with having versions. like `v2.5` and then update docs every new minor or major version gets released.

Reply:
@thomasvn, the problem here with listing files from the `develop` branch is we put users in a precarious and potentially unsupported position when they install a versioned release but use a values file which doesn't correspond to that release. We really need to get away from that practice entirely when we're talking about our official docs. Rather than putting in a placeholder like `<$VERSION>` we should make this the same as the version referenced by the docs. For example, if users are reading version 2.5 of our docs then any files that are associated with that release should be pointed to the `v2.5` branch. This obviously means that any new files that get referenced are "released" in that branch.